core/src/main/resources/messages/messages_en.yml

```
@@ -461,7 +461,7 @@ Particle-Effects:
     name: '&f&lAngel Wings'
     Description: '&7&oBecome an angel!'
   RainbowWings:
-    name: '&l&cR&6a&ei&an&9b&1o&dw&f Wings'
+    name: '&c&lR&6&la&e&li&a&ln&9&lb&1&lo&d&lw &f&lWings'
```

Should `Rainbow` be the same color order as the Projectile one that was created first (by you 😄)?
```suggestion
    name: '&4&lR&6&la&e&li&a&ln&b&lb&1&lo&5&lw &f&lWings'
```

---

Yes, fixed. 👍